packages/hooks/src/useTotalAccountBalance.ts

```diff
@@ -8,7 +8,7 @@ import useExchangeRate from './useExchangeRate';
  * first account in the list
  */
 
-const useTotalAccountBalance = (accounts: { balance?: number; currency?: string }[]) => {
+const useTotalAccountBalance = (accounts: { balance?: string | number; currency?: string }[]) => {
```

Comment:
This change was made to make CI pass, it was raising type error before.
Hope it won't result in a merge conflict later